### DISCUSSION TITLE
Intermittently noisy values

### DISCUSSION THREAD

**Discussion by canuckray** [emporia-vue-local/esphome#145] [Q&A]
Hello all. 

I am using a Gen 2 Emporia with Phase A and B measurement only (no individual circuit sensors).  I am experiencing noisy sensor values coming from the 2 sensors, but only some of the time.   Interestingly, as you can see on the plot below, the noise appears (and disappears) from the 2 phases at the same time.    I have confirmed the moving average filter is in tact on both sensors in esphome.  Has anyone experienced this artefact, or perhaps offer an explanation?  Is it likely a load within my home, a bad emporia unit, or other?

![image](https://user-images.githubusercontent.com/79683004/224801123-898ee12b-d23f-47d0-b44a-f1da1a6c9e2a.png)

**Comment by flaviut** [comment]
Never seen this before! I'd suggest checking a few things:

- what does your voltage graph look like?
- does this happen at the same time every day?
- are all the clamps fully closed? Are all the connectors fully inserted (I had trouble with mine staying in while I closed the panel)?
- If you have a multimeter, what does it say about the voltages? Does it agree with the vue?
